canairio/wsgi.py

```diff
@@ -11,6 +11,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'canairio.settings')
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'canairio.settings.production')
```

Comment:
This is generally not a good idea, we should be able to handle variable environment definitions within `settings.py` instead of having separate settings files for each environment.

Reply:
Yes, we can handle the most with env vars.  I'm not sure how can we handle with travis, testing, development and production.  If we are able to put all the required envvars is just perfect.